product_equipment/models/trust_crm_workflow.py

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
+# © 2016 Alessandro Fernandes Martini, Trustcode
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class CrmHelpdeskType(models.Model):
+    _name = "crm.helpdesk.type"
+    _order = 'sequence'
```

Comment:
meu brother... tem um bug na instalação do módulo

2016-08-25 17:23:54,695 812 ERROR alb2408 werkzeug: Error on request:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 180, in run_wsgi
    execute(self.server.app)
  File "/usr/lib/python2.7/dist-packages/werkzeug/serving.py", line 168, in execute
    application_iter = app(environ, start_response)
  File "/opt/odoo/OCB/openerp/service/server.py", line 291, in app
    return self.app(e, s)
  File "/opt/odoo/OCB/openerp/service/wsgi_server.py", line 216, in application
    return application_unproxied(environ, start_response)
  File "/opt/odoo/OCB/openerp/service/wsgi_server.py", line 202, in application_unproxied
    result = handler(environ, start_response)
  File "/opt/odoo/OCB/openerp/http.py", line 1297, in __call__
    return self.dispatch(environ, start_response)
  File "/opt/odoo/OCB/openerp/http.py", line 1271, in __call__
    return self.app(environ, start_wrapped)
  File "/usr/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 591, in __call__
    return self.app(environ, start_response)
  File "/opt/odoo/OCB/openerp/http.py", line 1271, in __call__
    return self.app(environ, start_wrapped)
  File "/usr/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 591, in __call__
    return self.app(environ, start_response)
  File "/opt/odoo/OCB/openerp/http.py", line 1271, in __call__
    return self.app(environ, start_wrapped)
  File "/usr/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 591, in __call__
    return self.app(environ, start_response)
  File "/opt/odoo/OCB/openerp/http.py", line 1271, in __call__
    return self.app(environ, start_wrapped)
  File "/usr/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 591, in __call__
    return self.app(environ, start_response)
  File "/opt/odoo/OCB/openerp/http.py", line 1271, in __call__
    return self.app(environ, start_wrapped)
  File "/usr/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 591, in __call__
    return self.app(environ, start_response)
  File "/opt/odoo/OCB/openerp/http.py", line 1435, in dispatch
    ir_http = request.registry['ir.http']
  File "/opt/odoo/OCB/openerp/http.py", line 353, in registry
    return openerp.modules.registry.RegistryManager.get(self.db) if self.db else None
  File "/opt/odoo/OCB/openerp/modules/registry.py", line 339, in get
    update_module)
  File "/opt/odoo/OCB/openerp/modules/registry.py", line 370, in new
    openerp.modules.load_modules(registry._db, force_demo, status, update_module)
  File "/opt/odoo/OCB/openerp/modules/loading.py", line 350, in load_modules
    force, status, report, loaded_modules, update_module)
  File "/opt/odoo/OCB/openerp/modules/loading.py", line 255, in load_marked_modules
    loaded, processed = load_module_graph(cr, graph, progressdict, report=report, skip_modules=loaded_modules, perform_checks=perform_checks)
  File "/opt/odoo/OCB/openerp/modules/loading.py", line 157, in load_module_graph
    init_module_models(cr, package.name, models)
  File "/opt/odoo/OCB/openerp/modules/module.py", line 286, in init_module_models
    result = obj._auto_init(cr, {'module': module_name})
  File "/opt/odoo/OCB/openerp/api.py", line 268, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/OCB/openerp/models.py", line 2639, in _auto_init
    self._set_default_value_on_column(cr, k, context=context)
  File "/opt/odoo/OCB/openerp/api.py", line 268, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/OCB/openerp/models.py", line 2401, in _set_default_value_on_column
    default = default(self, cr, SUPERUSER_ID, context)
  File "/opt/odoo/OCB/openerp/api.py", line 268, in wrapper
    return old_api(self, *args, **kwargs)
  File "/opt/odoo/OCB/openerp/api.py", line 372, in old_api
    result = method(recs, *args, **kwargs)
  File "/opt/odoo/OCB/openerp/fields.py", line 432, in <lambda>
    lambda recs: self.convert_to_write(value(recs))
  File "/opt/odoo/trust-addons/product_equipment/models/trust_crm_workflow.py", line 25, in _default_stage_id
    return stage[0]
  File "/opt/odoo/OCB/openerp/models.py", line 5679, in __getitem__
    return self._browse(self.env, (self._ids[key],))
IndexError: tuple index out of range